test/utils/base_page/base_page.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('../../../test/config/logger.config.js');
+const Header = require('./header');
+
+class BasePage {
+    constructor() {
+        this.Header = new Header();
+    };
+    wait(seconds) {
+        logger.info(`Waiting "${seconds * 1000}" milliseconds`);
+        return browser.sleep(seconds * 1000);
+    };
+    async getCurrenUrl() {
```

Comment:
Please move `getCurrentUrl()` method to a separate function helper.

Reply:
done